test/dynamo/test_unspec.py

```diff
@@ -433,6 +433,15 @@ def fn(x):
         self.assertExpectedInline(cnts.frame_count, """2""")
         self.assertExpectedInline(cnts.op_count, """3""")
 
+    @torch._dynamo.config.patch(capture_scalar_outputs=True)
+    def test_split_aot_autograd(self):
+        @torch.compile(backend="aot_eager", fullgraph=True)
```

Comment:
Not for this PR, but there is kind of a lack of good places where we test backend="aot_eager", I have an idea, will put up a PR exploring it shortly. Probably making the backend slug an input to the test, and then generating dynamo,aot_eager,inductor versions.